src/ui/components/NetworkMonitor/NetworkMonitorListRow.module.css

```diff
@@ -124,7 +124,7 @@
 
 .TimingContainer {
   position: relative;
-  height: 0.75rem;
+  height: 0.8rem;
```

Comment:
IMO 0.8 rem is not a very useful value. Might as well use pixels if we aren't sticking to the same increments.

Reply:
We might want to have a chat about this to get on the same page. I definitely optimise for the final visual, and .25 sometimes causes a misalignment. In this case, .75 was causing the top and bottom margin to feel optically uneven, which lead to us using 27 height (rather than the 26 I was aiming for standardising on)

I'm happy to use whatever units we want to achieve this effect. Are you saying you'd prefer pixels when I need to do this fine-tuning pixel work?

Reply:
Yes.